### ISSUE TITLE
Further CSS tidy up

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Status#18]
- Fix order of CSS imported in from the rules we used from Bootstrap and the modern.ie header
- Improve CSS used by the header
- Remove reduced bootstrap.css from the repo
- Remove any declarations left over that are never used (always overridden or same as one that is inherited)
- Remove units from 0 lengths that still have them
- Remove any duplicate declarations
- Add any fallbacks or vendor prefixes that could be useful
- Refactor any declarations that we continuously set
- Look into CSS Lint and similar tools to see if we’re missing anything
- Look for any CSS rendering bottlenecks

**Comment by dstorey** [comment]
Started work on this in branch css-tidy-up

**Comment by dstorey** [comment]
I’ve pushed the commits which resolve this to the development branch. It needs more testing to make absolute sure as I changed a lot of the CSS. One downside is we lost the animation on hover for the Twitter logo when changing to an SVG file (browsers ignore the SVG hover state when used via CSS `background-image` For security reasons I believe?)

Only thing left from this bug is to look into improving the menu CSS. I refactored a lot from the modern.ie CSS, but still has a lot of positioning that can probably be removed.
